ADMIN.adoc

```diff
@@ -0,0 +1,36 @@
+= Repository administrator responsibilities
+
+== Administrator rotation schedule
+
+Every 2 weeks a different project member assumes the role of repository administrator.
+Signing up to be an administrator is voluntary.
+To volunteer to be an administrator, contact @cbudz or @bergerhoffer.
+
+== Administrator responsibilities
+
+As the administrator, you are the point of contact if any contributors experience issues with the repository.
+You also have the following responsibilities:
+
+* Driving resolution of open issues
+* Driving completion of pull requests
+* Maintaining the Style Guide site on GitHub
+
+=== Driving resolution of open issues
+
+* Review link:https://github.com/redhat-documentation/supplementary-style-guide/issues[issues] submitted to the supplementary style guide GitHub repository.
+* If none of the project members respond to a newly opened issue, add the _Needs Fix_ label to the issue and mention @redhat-documentation/ccs-style-council in a comment on the issue to notify project members about it.
```

Comment:
Maybe make @redhat-documentation/ccs-style-council a link to https://github.com/orgs/redhat-documentation/teams/ccs-style-council?

Reply:
Also, what is the "Needs fix" label for? I see that we don't have this label and I'm not sure I understand its purpose here. CC @cbudz

Reply:
After discussing, the Needs fix label isn't needed. CC @inoxx03

Reply:
Thank you @bergerhoffer. I'll update the PR and remove the line about the _Needs Fix_ label.